sdk/fluidrelay/Azure.ResourceManager.FluidRelay/api/Azure.ResourceManager.FluidRelay.netstandard2.0.cs

```diff
@@ -0,0 +1,163 @@
+namespace Azure.ResourceManager.FluidRelay
+{
+    public partial class FluidRelayContainerCollection : Azure.ResourceManager.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource>, System.Collections.IEnumerable
+    {
+        protected FluidRelayContainerCollection() { }
+        public virtual Azure.Response<bool> Exists(string fluidRelayContainerName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<bool>> ExistsAsync(string fluidRelayContainerName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource> Get(string fluidRelayContainerName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource> GetAll(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource> GetAllAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource>> GetAsync(string fluidRelayContainerName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        System.Collections.Generic.IAsyncEnumerator<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource> System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource>.GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken) { throw null; }
+        System.Collections.Generic.IEnumerator<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource>.GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public partial class FluidRelayContainerData : Azure.ResourceManager.Models.ResourceData
+    {
+        public FluidRelayContainerData() { }
+        public string FrsContainerId { get { throw null; } }
```

Comment:
Are `FrsContainerId` and `FrsTenantId` GUID?

Reply:
Confirming with service team

Reply:
Confirmed it's `guid`